src/libbson/src/bson/bson-context.c

```diff
@@ -43,49 +43,24 @@
  * Globals.
  */
 static bson_context_t gContextDefault;
+static int64_t gRandCounter = INT64_MIN;
```

Comment:
Can this variable be moved into `_bson_context_init_random()` as a static local variable?

I also recommend adding a comment to documentation its purpose.

Reply:
Yes